lstchain/mc/tests/test_senstivity.py

```diff
@@ -34,22 +33,17 @@ def test_read_sim_par(simulated_dl1_file):
 
 @pytest.mark.run(after='test_apply_models')
 def test_process_mc(simulated_dl2_file):
+    #Write some delta_t in the example dl2 file
+    dl2 = pd.read_hdf(simulated_dl2_file, key=dl2_params_lstcam_key)
+    dl2["delta_t"]=0.0005
```

Comment:
`simulated_dl2_file` has only one entry?

Reply:
No, but this way it writes the same value on all events which I think is ok for testing purposes.